doc/user/content/sql/drop-type.md

```diff
@@ -17,30 +17,88 @@ menu:
 Field | Use
 ------|-----
 _data_type_name_ | The name of the type to remove.
-`CASCADE` | Automatically drop objects that depend on the type (such as table columns, functions, operators, or other types)
+`CASCADE` | Automatically removes objects that depend on the type, such as tables or other types.
```

Comment:
Nit: maybe `such as tables or other objects` instead of `other types`.

Reply:
"Automatically removes **objects** that depend on the type, such as tables or other **objects**" is tautological. We can list other objects here to give the user an idea of the dependencies they need to be aware of, or we can cut the sentence off after "type".

What's the issue with "other types" here?

Reply:
I think `other types` also works! I just wanted it to be clear that `CASCADE` will drop any type of database object.

Reply:
Does "Automatically removes any object that depends on the type, such as tables or other types" better convey that we're listing examples but not every case? Or do you have a list of every object that might be affected?

Reply:
I like that proposal!